chat.go

```diff
@@ -78,9 +78,10 @@ func (c *Chat) Recipient() string {
 type ChatMember struct {
 	Rights
 
-	User  *User        `json:"user"`
-	Role  MemberStatus `json:"status"`
-	Title string       `json:"custom_title"`
+	User        *User        `json:"user"`
+	Role        MemberStatus `json:"status"`
+	Title       string       `json:"custom_title"`
+	IsAnonymous bool         `json:"is_anonymous"`
```

Comment:
As we named `is_anonymous` field `Anonymous` once, we should use this name for further similar fields to preserve consistency. https://github.com/tucnak/telebot/blob/62cfa356f7f6a2c9e0ad011c32f3df2c076a57b4/polls.go#L22